common/aws/ec2/ec2.go

```diff
@@ -43,47 +44,48 @@ type EC2Internal interface {
 }
 
 // https://aws.amazon.com/ec2/instance-types/
-var InstanceSizes = map[string]interface{}{
-	"t2.nano":     nil,
-	"t2.micro":    nil,
-	"t2.small":    nil,
-	"t2.medium":   nil,
-	"t2.large":    nil,
-	"m4.large":    nil,
-	"m4.xlarge":   nil,
-	"m4.2xlarge":  nil,
-	"m4.4xlarge":  nil,
-	"m4.10xlarge": nil,
-	"m3.medium":   nil,
-	"m3.large":    nil,
-	"m3.xlarge":   nil,
-	"m3.2xlarge":  nil,
-	"c4.large":    nil,
-	"c4.xlarge":   nil,
-	"c4.2xlarge":  nil,
-	"c4.4xlarge":  nil,
-	"c4.8xlarge":  nil,
-	"c3.large":    nil,
-	"c3.xlarge":   nil,
-	"c3.2xlarge":  nil,
-	"c3.4xlarge":  nil,
-	"c3.8xlarge":  nil,
-	"g2.2xlarge":  nil,
-	"g2.8xlarge":  nil,
-	"x1.32xlarge": nil,
-	"r3.large":    nil,
-	"r3.xlarge":   nil,
-	"r3.2xlarge":  nil,
-	"r3.4xlarge":  nil,
-	"r3.8xlarge":  nil,
-	"i2.xlarge":   nil,
-	"i2.2xlarge":  nil,
-	"i2.4xlarge":  nil,
-	"i2.8xlarge":  nil,
-	"d2.xlarge":   nil,
-	"d2.2xlarge":  nil,
-	"d2.4xlarge":  nil,
-	"d2.8xlarge":  nil,
+var InstanceSizes = map[string]bytesize.Bytesize{
+	"t2.nano":     0.5 * bytesize.GiB,
```

Comment:
this is interesting... what's the reason we need to track the (memory?) size of each instance?